tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -64,7 +95,10 @@ void SetUpExplorationTest()
     void SetUpExplorationTest_Debugger()
     {
         Logger.Information($"Set up exploration test for debugger.");
-        //TODO TBD
+        if (LineProbes)
+        {
+            CreateLineProbes();
+        }
```

Comment:
I don't get it. Where's the code that sets this flag to be `true` 🤔 have you verified it's running in the CI?

If it's not running in CI, then I would suggest to change `ExplorationTestDescription` class to include a `LineProbesEnabled` property.
Set it to `true` only when initializing the instance of protobuf. This way, we'll be able to run Protobuf with Method + Line probes while all the rest will still stay intact.
If you do that, you won't need the global `LineProbes` flag anymore, you'll need to simply refer to the property of the corresponding `ExplorationTestDescription`.